src/reports_api/services/code.py

```diff
@@ -29,9 +29,9 @@ def find_code_values_by_type(
         """Find code values by code type."""
         current_app.logger.debug(f'<find_code_values_by_type : {code_type}')
         model: CodeTable = cls._find_model_from_type(code_type)
-        response = {'codes':[]}
```

Comment:
What is this requirement to change this.

Reply:
It seemed not right to return work_types, phases, etc as `codes` instead of what they actually are. Just my personal preference, can revert back if there was some reason for doing it this way.

Reply:
Reverted